Makefile

```diff
@@ -240,7 +240,7 @@ ux-tarball: build-ux ## Build website asset tarball
 ##@ Test 🧪
 
 .PHONY: test
-test: test-unit test-integration test-e2e test-typecheck ## Run all tests
+test: test-unit test-integration test-e2e test-typecheck test-grapl-template-generator ## Run all tests
```

Comment:
tbh I think we should just remove this target; I don't think *anybody* uses it.

Reply:
I used it by accident today, it was a mistake. I'd be in favor of removing it.